src/music.cpp

```diff
@@ -0,0 +1,86 @@
+#include "music.h"
+#include <fstream>
+#include <iostream>
+
+Asound::Asound() {
+  std::ifstream file(MUSIC_CONFIG_PATH + "config.json");
+
+  if (!file.is_open()) {
+    std::cerr << "Failed to open config file for music" << std::endl;
+    exit(1);
+  }
+
+  file >> config;
+
+  for (auto &[k, v] : config.items()){
+    std::cout << "Read sound key: " << k << " val: " << v << std::endl;
+    std::string s = MUSIC_CONFIG_PATH + std::string(v);
+    sounds.emplace(k, s);
+  }
+}
+
+Asound::~Asound() {}
+
+void Asound::play(std::string sound){
```

Comment:
pass temporary object arguments like this as `const std::string &sound` to avoid unnecessary copy construction

Reply:
done